guides/predictive-analytics-with-deepnote.md

```diff
@@ -0,0 +1,116 @@
+---
+description: >-
+  Deepnote is a versatile notebook that revolutionizes analytics and data
+  science workflow by seamlessly combining the power of SQL and Python with a
+  wide range of data frameworks.
+---
+
+# Predictive analytics with Hydra using Deepnote
+
+The focus of this guide is to display the ease of integrating Hydra into [Deepnote](https://deepnote.com/docs) to perform analytics and predictions using Python scripts.
+
+A simple salary dataset was chosen with two columns 'Years of experience' and 'Salary' to make predictions.
+
+## Setup
+
+* Ensure you have the details of your Hydra instance handy by creating a data warehouse in Hydra as shown in the [Setup guide](https://docs.hydra.so/getting-started/setup-guide).
+* Sign up for your free instance at [Deepnote](https://deepnote.com/sign-up)
+
+### Building the table with Hydra
+
+1. Sample data from [Kaggle](https://www.kaggle.com/datasets/rsadiq/salary) would be used for this guide. Download the CSV file.
+2. Connect to your Hydra database and import the CSV file to the database using the steps for importing a CSV file outlined in the [Documentation](https://docs.hydra.so/centralize-data/load/from-local-csv-file)
+
+### Creating the Connection
+
+Now we can connect the Postgres Database to Deepnote
+
+1. Start by [Signing Up](https://deepnote.com/sign-up) or Logging in if you already have an account.
+2. Create a workspace name, in this Demo, it is called Hydra-Team, and choose the data source as PostgreSQL as shown in the image below.
```

Comment:
Edit to this sentence: Create a workspace name, which will be "Hydra-Team" in this demo. Next, choose the data source as PostgreSQL as shown in the image below.